pkg/csi/recover/register.go

```diff
@@ -18,12 +18,14 @@ package recover
 
 import (
 	"github.com/fluid-cloudnative/fluid/pkg/csi/config"
+	"github.com/fluid-cloudnative/fluid/pkg/csi/features"
+	utilfeature "github.com/fluid-cloudnative/fluid/pkg/utils/feature"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
 // Register initializes the fuse recover and registers it to the controller manager.
 func Register(mgr manager.Manager, config config.Config) error {
-	fuseRecover, err := NewFuseRecover(mgr.GetClient(), mgr.GetEventRecorderFor("FuseRecover"), config.RecoverFusePeriod)
+	fuseRecover, err := NewFuseRecover(mgr.GetClient(), mgr.GetEventRecorderFor("FuseRecover"))
```

Comment:
`config.RecoverFusePeriod` is no longer used by `NewFuseRecover`. I think we can remove it from `config.Config`